rfcs/README.md

```diff
@@ -0,0 +1,45 @@
+# GeckoView RFC Process
+
+Changes to the GeckoView API that impact consumers are considered "significant" changes. We try to make sure that our APIs are the best that they can be, and in order to ensure that significant changes must go through an RFC (Request For Comments) process. The process is here to provide structure, guidance and feedback on significant changes leading to APIs that are useful, understandable and easy to use.
+
+The RFC process allows feedback to be gathered from all interested parties on well thought through and explained proposals for change.
+
+If you want to make a change to GeckoViews APIs, if you can answer "yes" to any of these questions then your change is a "significant" change and you must submit an RFC before landing the change.
+
+- Does your change allow access to a new feature, or make public a feature previously only available internally?
+- Does your change change the behaviour of an existing API such that consumers would need to update their code in order to continue using the feature?
+- Does your change remove existing functionality from users requiring them to use alternative methods to continue providing the same functionality to their users?
+
+If you answered yes to any of these questions, please follow the RFC process outlined below to ensure that your change is accepted into GeckoView.
+
+## The RFC Process
+
+- Fork the GeckoView repository.
+- Make a copy of the [RFC template](rfcs/_templates/rfc-template-000.md). 
+- Rename the file to `<proposal-name>-0000.md`, where proposal-name is a short but descritive name for your proposal and save it under `rfcs/`. 
+- Fill in the RFC document. Please ensure to fill it in carefully and flesh out as much detail as possible. 
+    + You may wish to use the PR mechanism to gather early feedback from your team as a work in progress. If so, please add "WIP" to the front of the title of the pull request and add the "Work In Progress" label. Do not add any reviewers other than your team until your RFC is no longer a work in progress.
+- Submit a pull request containing your RFC and any supporting documentation.
+- Assign the PR to yourself.    
+    + If you are unable to assign the PR to yourself, add your details, including the team you are working on, to the description section of the PR.
+- Add the "RFC" label to the PR. Remove any "WIP" labels and title amendments.
+- Add a link to any associated Bugzilla or Github issue in the PR description.
+- Request review from the `GeckoView` team, plus any other interested parties as needed. Reach out to as many potential consumers of the API as you can. If your desired reviewer is not on GitHub, you can use the following methods to contact them:
+    + Email reviewer directly and request feedback giving link to PR.
```

Comment:
I don't have solutions but I just want to mention that in the messaging case we did all of these and nothing worked (as in, we got no traction until the code was written after waiting 3 weeks for feedback).

Having a living doc that we can easily reference and go back to for the most up-to-date version of the design will probably help though.

Reply:
@agi90 I think there comes a point where, if we need feedback from a critical reviewer and they are simply not responding then it needs to be escalated to a higher power in order to get it looked at (i.e. @davidb). 

It might be worth asking @kmag what you could have done to have gotten as response from him before the code dropped. He might be able to give us some insight into how to address the problem here.